chemicaltoolbox/chemfp/mol2fps.xml

```diff
@@ -3,8 +3,6 @@
     <!--parallelism method="multi" split_inputs="infile" split_mode="to_size" split_size="10000" shared_inputs="" merge_outputs="outfile"></parallelism-->
     <requirements>
         <requirement type="package" version="1.1p1">chemfp</requirement>
-        <requirement type="package" version="2016.03.3">rdkit</requirement>
-        <requirement type="package" version="2.3.2">openbabel</requirement>
```

Comment:
Why these dependencies was removed? I don't follow you in this, please, can you explain?